app/views/help/verbal_ntp_requests.md

```diff
@@ -0,0 +1,5 @@
+<% title "How do I submit an verbal NTP request in case of emergency?" %>
```

Comment:
Would they know what an "NTP" is? Or should this say "What do I do to submit an emergency request?" or something like that?

Also, this page is specific to NCR...not sure the best way to split them up.